browser/extensions/brave_extension_provider_browsertest.cc

```diff
@@ -65,6 +66,15 @@ IN_PROC_BROWSER_TEST_F(BraveExtensionProviderTest, PDFJSInstalls) {
       "window.domAutomationController.send("
         "!!document.body.querySelector(\"#chrome-pdfjs-logo-bg\"))", &pdfjs_exists));
   ASSERT_TRUE(pdfjs_exists);
+
+  // Ensure that the extension prefix is not in the display URL (brave-browser#368)
+  EXPECT_STREQ(contents->GetController().GetLastCommittedEntry()
+               ->GetVirtualURL().spec().c_str(),
```

Comment:
also nit: I would actually indent this 4 more spaces to show continuation of the current param vs next param below which can keep the current indentation